components/data-table/__tests__/data-table.browser-test.jsx

```diff
@@ -390,7 +390,6 @@ describe('DataTable: ', function() {
 			).call(this);
 
 			const thead = getTable(this.dom).querySelectorAll('thead')[0];
-			const thirdColumn = thead.querySelectorAll('th')[2];
```

Comment:
Similar unused var on this line:
https://github.com/salesforce/design-system-react/pull/1854/files#diff-e9ab4feff220627b3e05e39665df8a5aR418

Can you also get rid of it?

Reply:
@davidlygagnon I will remove it.